README.md

```diff
@@ -1,12 +1,14 @@
 # Pristine
 
-Pristine is an open source repository in its original condition. Its goal is to be a starting point for new open source projects following a Documentation Driven Development pattern, as well as a resource from which to augment existing documentation.
+Pristine is an open source repository in its original condition.
+
+There are a lack of repositories to start from to build community driven open source projects. Pristine is a starting point, it follows a Documentation Driven Development approach, and can be used as a resource to augment existing documentation.
```

Comment:
```suggestion
Pristine is a starting point, it follows a Documentation Driven Development approach, and can be used as a resource to augment existing documentation.
```

Not real keen personally on pointing to 'an absence of' here, since it's not mission critical for understanding what this project is about, and doesn't have backing data. (Or since there are [a lot of "boilerplate" repos](https://github.com/search?p=1&q=boilerplate+starter+repo&type=Repositories), speak to how this boilerplate differs).